src/Tests/SettingsDataTrait.php

```diff
@@ -30,11 +30,11 @@ private function setValidSettings() {
    */
   private function getValidCredentialSettings() {
     return [
-      'account_id' => 'account_id_1',
-      'site_id' => 'site_id_1',
-      'assets_url' => 'assets_url_1',
+      'account_id' => 'AccountId1',
+      'site_id' => 'SiteId1',
+      'assets_url' => 'AssetsUrl1',
```

Comment:
"account_id_1" and "site_id_1" are no longer valid Account ID and Site ID, now we have string validation.